ui/authorization-manager-panel.reel/authorization-manager-panel.js

```diff
@@ -103,6 +124,18 @@ exports.AuthorizationManagerPanel = Component.specialize({
         }
     }
 
+}, {
+
+    instance: {
```

Comment:
"instance" is technically correct, but not a great API name

Reply:
https://github.com/montagejs/montage/commit/aeb0e486c3006a42d095f76ad7038d17a39c6de9